### PR TITLE
Change CD Visibility documentation based on service-centric view

### PR DESCRIPTION
**Pull request by andrea-mosk**
### What does this PR do? What is the motivation?

We recently changed several UI pages from a deployment-centric approach to a service-centric approach. This PR updates the description of the view and includes the new screenshots

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/continuous_delivery/search/_index.md [hunk 1]
```diff
@@ -15,44 +15,42 @@ further_reading:
...
 
-{{< img src="/continuous_delivery/search/deployments.png" text="The Deployments page in Datadog" style="width:100%" >}}
+The [**Deployments** page][1] shows stats aggregated by services and environments over the selected time frame, as well as the status of the latest deployment execution. Use this page to see all your service deployments and get a view of their health.
+The metrics shown include the number of executions and failures, the failure rate, the median duration, and the 95th percentile duration. This information reveals which deployments have a higher probability of failure and which deployments are taking the most time to be executed. The effect of the latest changes can be seen by checking the status, revision and time of the last deployment result.
 
-Access high-level accumulation and trends, including:
+<div class="alert alert-info">Deployments without services configured and partial deployment executions are excluded from the statistics aggregation of the Deployments page. You can search for these deployments in the Deployment Executions page: <code>@deployment.partial_deployment:* OR -@deployment.service:*</code>.</div>
```
Comment (maycmlee) [inline content/en/continuous_delivery/search/_index.md:25]: In the first sentence, is it deployments with partial deployment executions that are excluded or deployments without partial deployment executions that are excluded?

File: content/en/continuous_delivery/search/_index.md [hunk 2]
```diff
@@ -15,44 +15,42 @@ further_reading:
...
+The **Deployment** page provides you high-level information, including:
 
-Metrics shown include the number of executions and failures, the failure rate, the median duration, and the 95th percentile duration. This information reveals which deployments are high-usage and potentially high resource consumers. The last deployment result, duration, and last status shows you the effect of the latest changes.
-
-<div class="alert alert-info">Partial deployment executions are excluded from the statistics aggregation to avoid incorrect measures. You can use the Partial Deployment facet to search for these deployments: <code>@deployment.partial_deployment:*</code>.</div>
+- An overview of the health of the different services and environments, with aggregated stats.
+- A window for spotting and fixing immediate, urgent issues like broken deployments in production.
+- How each service deployment was executed, over time, and with what results and trends.
```
Comment (maycmlee) [inline content/en/continuous_delivery/search/_index.md:35]:
```suggestion
- How each service deployment was executed, over time, and the results and trends.
```

**Review by maycmlee — COMMENTED** [review]
Just a small suggestion and a clarifying question, otherwise looks great!

File: content/en/continuous_delivery/search/_index.md (hunk 1 above)
Comment (andrea-mosk) [reply content/en/continuous_delivery/search/_index.md:25]: There are two types of deployment excluded from the page:
1. Partial deployments (i.e. only a subset of steps were executed for a deployment)
2. Deployments without service configured (they might or might not have a service, and in the latter case they are excluded from the aggregated stats)

Comment (maycmlee) [reply content/en/continuous_delivery/search/_index.md:25]: @andrea-mosk Thanks for the explanation! Does this work?

> Deployments with no services configured and those with partial deployment executions are excluded from the statistics aggregation of the Deployments page.

Comment (andrea-mosk) [reply content/en/continuous_delivery/search/_index.md:25]: Yeah looks good, I would just remove "those that":

> Deployments with no services configured and partial deployment executions are excluded from the statistics aggregation of the Deployments page.

Comment (andrea-mosk) [reply content/en/continuous_delivery/search/_index.md]: Changed 😃

**Review by maycmlee — APPROVED** [review]
@andrea-mosk LGTM! Let me know if this is go to merge.

**Comment by andrea-mosk** [comment]
> @andrea-mosk LGTM! Let me know if this is go to merge.

@maycmlee This is good to go! :)

**Pull request merged**